core/templates/services/playthrough-issues.service.ts

```diff
@@ -96,7 +96,7 @@ angular.module('oppia').factory('PlaythroughIssuesService', [
           explorationId, explorationVersion);
       },
       getPlaythrough: function(playthroughId) {
```

Comment:
`getPlaythroughAsync: async function(`*

Reply:
Um, sorry for a stupid doubt but is _getPlaythrough_ also an async function? and do I have to change its name everywhere else as well?

Reply:
> getPlaythrough also an async function?

It returns PlaythroughIssuesBackendApiService.fetchPlaythrough(...) and `PlaythroughIssuesBackendApiService.fetchPlaythrough(...)` function returns a promise, right?

> and do I have to change its name everywhere else as well?

Yep! :)